### PR TITLE
Note that `ninja = false` goes under `[llvm]`

### PR DESCRIPTION
**Pull request by jyn514**
Addresses https://github.com/rust-lang/rust/issues/84938#issuecomment-852448332 - @kornelski does this look good?

r? @Mark-Simulacrum cc @joshtriplett

### CODE REVIEW

File: src/bootstrap/lib.rs [hunk 1]
```diff
@@ -1366,7 +1366,7 @@ impl Build {
                 eprintln!(
                     "
 Couldn't find required command: ninja
-You should install ninja, or set ninja=false in config.toml
+You should install ninja, or set ninja=false in config.toml under the [llvm] section.
```
Comment (joshtriplett) [inline src/bootstrap/lib.rs:1369]:
```suggestion
You should install ninja, or set `ninja=false` in config.toml in the `[llvm]` section.
```

This should either be "under the `[llvm]` section header" or "in the `[llvm]` section", and I think the latter is clearer.

**Comment by joshtriplett** [comment]
LGTM. r=me when ready once you have any other feedback you're seeking.

**Comment by jyn514** [comment]
Thanks for the review! This is small enough I think any further feedback can go in another PR.

@bors r=joshtriplett

**Pull request merged**